deployment/hasura/metadata/databases/AerieMerlin/tables/metadata/users_allowed_roles.yaml

```diff
@@ -0,0 +1,16 @@
+table:
+  name: users_allowed_roles
+  schema: metadata
+configuration:
+  custom_name: "users_allowed_roles"
+select_permissions:
+  - role: user
+    permission:
+      columns: [user_id, allowed_role]
+      filter: {"user_id":{"_eq":"X-Hasura-User-Id"}}
```

Comment:
I'm noticing that the yaml files capitalize session variable names in Hyphenated-Title-Case while the Gateway and hasura functions use kebab-case. Not a concern, just an observation - my guess is that the yaml version is case insensitive, while the SQL version is case sensitive

Reply:
Actually, it might be Hasura that's case-insensitive, as capitalization for the session variables in the yaml came straight out of the Hasura Permissions Builder UI